### PR TITLE
Add rpc_uri to sign_bid signature

### PR DESCRIPTION
**Pull request by mdantonio**
Additional, optional, parameter needed to support Thetanuts workflow

### CODE REVIEW

File: sdk_commons/sdk_commons/config.py [hunk 1]
```diff
@@ -126,6 +126,7 @@ def sign_bid(
         *,
         contract_address: str,
         chain_id: Chains,
+        rpc_uri: str,
```
[the remark this answers is not in the document]
Comment (mdantonio) [reply sdk_commons/sdk_commons/config.py:129]: No, this value is expected to be optional at implementation level, but if defined it is expected to be a `str`, not optional str.
At implementation level is optional intending that can be ignored and consumed as kwargs (i.e. we are only adding this parameter to Thetanuts)

[the remark this answers is not in the document]
Comment (pdonorio) [reply sdk_commons/sdk_commons/config.py:129]: Shouldn't it be the last one, then?

[the remark this answers is not in the document]
Comment (mdantonio) [reply sdk_commons/sdk_commons/config.py:129]: It's an option,
I currently added it at the same position as the other methods for consistency (all have `rpc_uri` after `chain_id`).
Also the abstract method is not aware of which of the parameters will be used or not by the concrete implementations, so should not consider this specific parameter in a special way

[the remark this answers is not in the document]
Comment (pdonorio) [reply sdk_commons/sdk_commons/config.py:129]: Thanks for clarifying

**Pull request merged**